webapp/publisher/snaps/views.py

```diff
@@ -51,6 +53,10 @@
 )
 
 store_api = SnapcraftStoreApi(talisker.requests.get_session(requests.Session))
+github = GithubAPI(
```

Comment:
Hmm it is a shame to be duplicating this block between here and the `oauth_views.py` file. Can you try to think of a way around that?